Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java

```diff
@@ -0,0 +1,89 @@
+package com.dwg_karrier.roys;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class ListActivity extends AppCompatActivity {
+  ListView lv;
+  ArrayList<ScriptedData> data;
+  Date finTime; // expected finish time
+  Date curTime; // current time
+  long duration; // time duration between current_time and finish time
+
+  protected void onCreate (Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.list);
+
+    final int minute = 60000;
+    final Intent intent = new Intent(this.getIntent());
+    finTime = (Date)intent.getSerializableExtra("finTime");
+    curTime = (Date)intent.getSerializableExtra("curTime");
+
+    duration = (finTime.getTime() - curTime.getTime()) / minute;
+    data = getData();
+
+    lv = (ListView)findViewById(R.id.listView);
+    lv.setAdapter(new ListViewAdapter(this, R.layout.item, data));
+
+    lv.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+      @Override
+      public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        Intent intent1 = new Intent(ListActivity.this, ContentView.class);
+        intent1.putExtra("finTime", finTime);
+        intent1.putExtra("curTime", curTime);
+        startActivity(intent1);
+        finish();
+      }
+    });
+  }
+
+  private ArrayList<ScriptedData> getData() {
+    ArrayList ret = new ArrayList<ScriptedData>();
+
+    /*url list 불러오는 함수
+     *ret = callurl();
+     */
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.8.6-comments
nit: space between /* and contents.